README.md

```diff
@@ -3,3 +3,41 @@
 This project is part of ASP.NET Core. You can find samples, documentation and getting started instructions for ASP.NET Core at https://github.com/aspnet/AspNetCore.
 
 Use https://github.com/aspnet/AspNetCore/issues for issues with this project.
+
+### Build on Windows ###
+```powershell
+PS> git submodule update --init
+PS> .\submodules\vcpkg\bootstrap-vcpkg.bat
+PS> .\submodules\vcpkg\vcpkg.exe install cpprestsdk cpprestsdk:x64-windows
```

Comment:
Since the below cmake call is only x64 you can remove `cpprestsdk` from this line, which should cut your build times in half by not installing the x86 version.